### PR TITLE
fix: reduced chunk size in chunking algorithms by 12 bytes

### PR DESCRIPTION
**Pull request by rapaktech**

### CODE REVIEW

**Review by TimoGlastra — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/reactnativebledidcomm/central/CentralManager.kt [hunk 1]
```diff
@@ -91,7 +91,7 @@ class CentralManager(private val context: ReactContext) {
 
         Thread {
             isSending = true
-            val chunkSize = min(connectedMtu, message.count())
+            val chunkSize = min(connectedMtu - 12, message.count())
```
[the remark this answers is not in the document]
Comment (rapaktech) [reply android/src/main/java/com/reactnativebledidcomm/central/CentralManager.kt:94]: Yep. Makes sense. On it!

**Comment by rapaktech** [comment]
@blu3beri I think I got it right for `android`, but I don't know for `ios`, and I have no way to test it

**Review by blu3beri — APPROVED** [review]
It looks good to me, but it must be tested before it is released. You can you use the example app inside the library if needed.

I think if long as it compiles it is okay for iOS.

**Comment by blu3beri** [comment]
Some older iOS devices support 20 bytes transport as default. I highly doubt this will cause any issues here, but moving that to 8 bytes is a tad bit low. This comment is more of a note, for potential issues in the future.

**Comment by rapaktech** [comment]
I'm setting up testing for android now. I will need help testing ios. @morrieinmaas can you help test cross-platform and iOS-iOS?

**Comment by morrieinmaas** [comment]
> @blu3beri I think I got it right for `android`, but I don't know for `ios`, and I have no way to test it

I can test this tomorrow with iOS <-> Android and iOS <-> iOS if no one else does meanwhile

**Comment by morrieinmaas** [comment]
Nice one @rapaktech! 🤞 this also just works on ios

**Comment by rapaktech** [comment]
Android builds and android-android works bidirectionally
Screenshot 2023-03-15 at 3 25 09 PM
Screenshot 2023-03-15 at 3 25 30 PM

**Review by TimoGlastra — APPROVED** [review]

**Comment by rapaktech** [comment]
Woohoo!

**Comment by morrieinmaas** [comment]
iOS <-> iOS worx

**Comment by morrieinmaas** [comment]
iOS<->Android works both ways

**Pull request merged**